src/content/reference/manifest.mdx

```diff
@@ -560,6 +560,47 @@ selector:
   app.kubernetes.io/name: vote
 ```
 
+### services ([object], optional)
+
+A list of other containers (other than the main development container) that you want to put on developer mode.
```

Comment:
If the key is called `services`, I think we need to call them `services`in the description no? 

```suggestion
A list of services that you want to put on developer mode along your your development container.
```